gapis/api/vulkan/replay.go

```diff
@@ -919,6 +920,10 @@ func (a API) Replay(
 		case timestampsRequest:
 			var numInitialCommands int
 			var err error
+			if exporter == nil {
```

Comment:
The exporter should probably be optional here, depending on if the request asked for handle mappings.